server/models/User.js

```diff
@@ -4,22 +4,26 @@ const bcrypt = require("bcryptjs");
 const userSchema = new mongoose.Schema({
   username: {
     type: String,
-    required: true,
-    unique: true
+    unique: true,
   },
   email: {
     type: String,
-    required: true,
-    unique: true
+  },
+  gmail: {
+    type: String,
+    unique: true,
+  },
```

Comment:
As mentioned above, I think just for better reading this could be `email`. If in the future we wanted to roll out email/password authentication then this would be odd as it would assume all emails are `gmail`

Reply:
fix gmail and user model eb6556e9d2872e75897e1501c59377847cc3ebe9